### PR TITLE
Update token after token expiry

### PR DESCRIPTION
**Pull request by shekhar-j**
# Description
When JMS client gets expired during polling, token needs to be updated. 

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| |

# Checklist:

- [X] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [X] I have verified that new and existing unit tests pass locally with my changes
- [X] I have not allowed coverage numbers to degenerate
- [X] I have maintained at least 80% code coverage
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] Backward compatibility is not broken

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

- [ ] Acceptance tests

### CODE REVIEW

File: apex/helper/poller.go [hunk 1]
```diff
@@ -25,6 +25,8 @@ import (
 	"net/http"
 
 	client "dell/apex-job-client"
```
Comment (doriac11) [inline apex/helper/poller.go:27]: Does this need to change to jsmClient if you are changing the name below?

**Review by doriac11 — APPROVED** [review]

**Review by nikitajoshi1 — APPROVED** [review]

**Comment by shekhar-j** [comment]
run e2e test

**Comment by shekhar-j** [comment]
run test e2e

**Comment by nikitajoshi1** [comment]
run e2e test

**Review by doriac11 — APPROVED** [review]

**Comment by shekhar-j** [comment]
run e2e test

**Review by doriac11 — APPROVED** [review]

**Review by nikitajoshi1 — APPROVED** [review]

**Pull request merged**